addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/internal/discovery/MinecraftDiscoveryService.java

```diff
@@ -78,6 +84,12 @@ private void discoverServers() {
         subscription.add(signsSubscription);
     }
 
+    @Modified
+    @Override
```

Comment:
@wborn why?

Reply:
It was also part of the [original XML](https://github.com/openhab/openhab2-addons/blob/242c956d4e32a3cd240c558109f65da243245e37/addons/binding/org.openhab.binding.minecraft/OSGI-INF/MinecraftDiscovery.xml#L12). 
So I added the annotation for generating equivalent XML.

Perhaps @ibaton knows if it was intentionally added to the orignal XML?

Reply:
I don't think it was intended, I have no memory of adding it intentionally.

Reply:
Thanks for the info! I've removed the `modified(..)` method and annotation in a new commit.